api/config/dev.exs

```diff
@@ -0,0 +1,66 @@
+import Config
+
+# Configure your database
+config :api, Api.Repo,
+  username: "postgres",
+  password: "postgres",
+  hostname: "localhost",
+  database: "performance_manager",
+  port: "5432",
```

Comment:
this is all the default config minus this bit for connectiong to our local database created by the docker-compose file at root.

it would be nice if this could be created directly from the .env file also at root.